lib/payload/scanner/SSL_TLS/engine.py

```diff
@@ -0,0 +1,674 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Pradeep Jairamani (github.com/pradeepjairamani)
+
+import os, time
+import socket
+import struct
+import re
+from sets import Set
```

Comment:
sets library is deprecated. You can directly use set as `a = set()`. Please make the corresponding changes